src/sidebar/services/annotation-activity.js

```diff
@@ -45,16 +46,16 @@ export class AnnotationActivityService {
       date: activityDate,
       annotation: {
         id: annotation.id,
+        isShared: _isShared(annotation.permissions),
```

Comment:
It could potentially be confusing that the `annotation` property differs from the API `Annotation` object here, instead of being a strict subset of it. However using `isShared` is much easier for the consumer to handle. Really we probably ought to add that to the h API object.